### ISSUE TITLE
Unable to create calendar (if there is no existing sources)

### ISSUE THREAD

**Issue by zvs001** [wmcmahan/react-native-calendar-events#285]
Code example:
```
  calendarId = await RNCalendarEvents.saveCalendar({
    name: CALENDAR_NAME,
    title: CALENDAR_NAME,
    accessLevel: 'owner',
    color: '#EEAE0F',
    source: {
      name: calendarSource,
      isLocalAccount: true,
    },
    entityType: 'event',
    ownerAccount: 'xxx@gmail.com',
  })
```

It was working for a while. 
Last time we started to get errors from IOS 13+ users.
Error: `no source found to create the calendar (local & icloud)`
Probably exists some SDK for source creation?

Customer with such issue sent screen of calendars page:

![issue](https://user-images.githubusercontent.com/25595191/69355691-e548e000-0c8a-11ea-9953-cb3a093a14b6.jpg)

I reproduce issue by toggling `Calendar` in iCloud settings.
My `On Phone` section disappeared also.

**Comment by kam89** [comment]
Hi there,

How about this?

```
const source = {
   name: 'xxx@gmail.com',
   type: 'LOCAL'
}

calendarId = await RNCalendarEvents.saveCalendar({
    name: CALENDAR_NAME,
    title: CALENDAR_NAME,
    accessLevel: 'owner',
    color: '#EEAE0F',
    source,
    entityType: 'event',
    ownerAccount: source.name,
  })
```

**Comment by zvs001** [comment]
@kam89 Hi. 
It doesn't help. I tried also few other... but not. :(
Also documentation says that `source` field for Android only.

**Comment by kam89** [comment]
Hi @zvs001, you are right. That source field is for Android only.

Do you check whether permission is granted?
[RNCalendarEvents.authorizationStatus()](https://github.com/wmcmahan/react-native-calendar-events#authorizationstatus)

**Comment by zvs001** [comment]
Of course. Everything is fine.
Library usually save calendar to `Default` source. 
The issue appears for IOS 13+ only. When this source is not exists for some users (see screenshot).  Issue appears for most users after they upgrade IOS. 
It can start work when you enable iCloud for calendars. But it is not a solution

**Comment by MoOx** [comment]
When you want to save a calendar you need to do this in a existing, valid source (using `findCalendars`).

**Comment by zvs001** [comment]
As I remember it was bug in IOS 13 and it started +/- working later.
I remember that user had to enable calendar for iCloud, because there was no valid sources.

Not sure what situation we have with recent versions.

**Comment by MoOx** [comment]
Situation might be the same, we didn't really pushed any change on that topic.

**Comment by filipef101** [comment]
https://github.com/wmcmahan/react-native-calendar-events/pull/347 Should address this.
Also `findCalendars` returns calendars, not sources so it is unrelated.

**Comment by MoOx** [comment]
Closed by #347
